### DISCUSSION TITLE
Update by "Name" supported ?

### DISCUSSION THREAD

**Discussion by Vincent-HD** [Devolutions/UniGetUI#2137] [Q&A]
Hi !
Sometimes I can't update packages because for an obscure reason I get a `No installed package found matching input criteria.`.
And it seems that when updating by name and not by id "solves" the problem, see https://github.com/microsoft/winget-cli/issues/2426

I'm wondering if there is an option somewhere to use package's name instead of id ?

thanks !

**Comment by marticliment** [comment]
I am sorry, but this is not possible right now. Some improvements on the WinGet integration are on the way, and this issue should be fixed by them
